minicluster/src/main/java/alluxio/multi/process/Clients.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.multi.process;
+
+import alluxio.client.MetaMasterClient;
+import alluxio.client.block.BlockMasterClient;
+import alluxio.client.file.FileSystem;
+import alluxio.client.file.FileSystemMasterClient;
+
+/**
+ * Container for various Alluxio clients.
+ */
+public final class Clients {
```

Comment:
Could we name the methods like `getFileSystemClient`, `getFileSystemMasterClient` ...?

Reply:
done